straitlets/ext/click.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+
+from operator import itemgetter
+
+import click
```

Comment:
I think that if I write:

```python
from straitlets.ext.click import JsonConfigFile
```
and get `ImportError: No module named 'click'` I will understand what is wrong.